src/handlers/admin/menu_handler.py

```diff
@@ -10,187 +11,103 @@
-                    callback_data=json.dumps(
-                        {
-                            "chat_id": selected_chat_id,
-                            "action": constants.Actions.set_kick_timeout,
-                        }
-                    ),
-                )
-            ],
-            [
-                InlineKeyboardButton(
-                    "Изменить сообщение при входе в чат",
-                    callback_data=json.dumps(
-                        {
-                            "chat_id": selected_chat_id,
-                            "action": constants.Actions.set_on_new_chat_member_message_response,
-                        }
-                    ),
-                )
-            ],
-            [
-                InlineKeyboardButton(
-                    "Изменить сообщение при перезаходе в чат",
-                    callback_data=json.dumps(
-                        {
-                            "chat_id": selected_chat_id,
-                            "action": constants.Actions.set_on_known_new_chat_member_message_response,
-                        }
-                    ),
-                )
-            ],
-            [
-                InlineKeyboardButton(
-                    "Изменить сообщение после успешного представления",
-                    callback_data=json.dumps(
-                        {
-                            "chat_id": selected_chat_id,
-                            "action": constants.Actions.set_on_successful_introducion_response,
-                        }
-                    ),
-                )
-            ],
-            [
-                InlineKeyboardButton(
-                    "Изменить сообщение напоминания",
-                    callback_data=json.dumps(
-                        {
-                            "chat_id": selected_chat_id,
-                            "action": constants.Actions.set_notify_message,
-                        }
-                    ),
-                )
-            ],
-            [
-                InlineKeyboardButton(
-                    "Изменить сообщение после кика",
-                    callback_data=json.dumps(
-                        {
-                            "chat_id": selected_chat_id,
-                            "action": constants.Actions.set_on_kick_message,
-                        }
-                    ),
-                )
-            ],
-            [
-                InlineKeyboardButton(
-                    "Получить текущие настройки",
-                    callback_data=json.dumps(
-                        {
-                            "chat_id": selected_chat_id,
-                            "action": constants.Actions.get_current_settings,
-                        }
-                    ),
-                )
-            ],
+def new_button(button_name: str, chat_id, action):
+    return InlineKeyboardButton(button_name, callback_data=json.dumps({"chat_id": chat_id, "action": action,}))
```

Comment:
I remember we discussed that but -- it seems that passing any object (i.e. dict) here is possible as long as we tell the bot to work properly with "arbitrary callback data" by setting `arbitrary_callback_data=True` for the `Updater` instance in `app.py`, [source](https://github.com/python-telegram-bot/v13.x-wiki/wiki/Arbitrary-callback_data).